addons/mass_mailing/models/mailing.py

```diff
@@ -353,10 +363,18 @@ def action_test(self):
         }
 
     def action_launch(self):
+        self.ensure_one()
```

Comment:
Actually ensure one is not required. Could work in batch without any modification as far as I understand code.

Reply:
That is correct, I don't think that it is required.